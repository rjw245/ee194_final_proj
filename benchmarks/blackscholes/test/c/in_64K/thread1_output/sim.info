{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread1_output',
             '-n',
             '8',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../../parsec-3.0/pkgs/apps/blackscholes/inst/amd64-linux.gcc-serial/bin/blackscholes',
             '1',
             '../../../inputs/in_64K.txt',
             'thread1_output/output.txt'],
 'git_revision': '',
 'host': 'dell26',
 'pin_version': 71313,
 'rusage': (11874.004876,
            6.137067,
            131464,
            0,
            0,
            0,
            199378,
            2,
            0,
            2664,
            14768,
            0,
            0,
            0,
            611683,
            1368527),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=8 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/c/in_64K/thread1_output --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../parsec-3.0/pkgs/apps/blackscholes/inst/amd64-linux.gcc-serial/bin/blackscholes 1 ../../../inputs/in_64K.txt thread1_output/output.txt'],
 't_elapsed': 13528.631680965424,
 't_start': 1461613543.038568,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 769568768L}
