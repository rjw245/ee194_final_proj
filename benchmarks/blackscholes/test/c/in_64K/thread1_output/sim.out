                                     | Core 0      | Core 1      | Core 2      | Core 3      | Core 4      | Core 5      | Core 6      | Core 7     
  Instructions                       |  5486152950 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  Cycles                             |  3760811474 |  3760811474 |  3760811474 |  3760811474 |  3760811474 |  3760811474 |  3760811474 |  3760811474
  IPC                                |        1.46 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  Time (ns)                          |  1413838900 |  1413838900 |  1413838900 |  1413838900 |  1413838900 |  1413838900 |  1413838900 |  1413838900
  Idle time (ns)                     |        3300 |  1413838900 |  1413838900 |  1413838900 |  1413838900 |  1413838900 |  1413838900 |  1413838900
  Idle time (%)                      |        0.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0% |      100.0%
Branch predictor stats               |             |             |             |             |             |             |             |            
  num correct                        |   396990098 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num incorrect                      |    14840080 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  misprediction rate                 |       3.60% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
  mpki                               |        2.71 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
TLB Summary                          |             |             |             |             |             |             |             |            
  I-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |   656674231 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num misses                       |          99 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |  1774824362 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num misses                       |    28967768 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       1.63% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        5.28 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  L2 TLB                             |             |             |             |             |             |             |             |            
    num accesses                     |    28967867 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num misses                       |      216346 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.75% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.04 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
Cache Summary                        |             |             |             |             |             |             |             |            
  Cache L1-I                         |             |             |             |             |             |             |             |            
    num cache accesses               |   656674231 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |      498663 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.08% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |        0.09 |         inf |         inf |         inf |         inf |         inf |         inf |         inf
  Cache L1-D                         |             |             |             |             |             |             |             |            
    num cache accesses               |  1774824362 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |     5001312 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       0.28% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |        0.91 |         inf |         inf |         inf |         inf |         inf |         inf |         inf
  Cache L2                           |             |             |             |             |             |             |             |            
    num cache accesses               |     5500024 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |     3042163 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |      55.31% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |        0.55 |         inf |         inf |         inf |         inf |         inf |         inf |         inf
  Cache L3                           |             |             |             |             |             |             |             |            
    num cache accesses               |     3042220 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    num cache misses                 |       71695 |           0 |           0 |           0 |           0 |           0 |           0 |           0
    miss rate                        |       2.36% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf% |        inf%
    mpki                             |        0.01 |         inf |         inf |         inf |         inf |         inf |         inf |         inf
DRAM summary                         |             |             |             |             |             |             |             |            
  num dram accesses                  |       35849 |           0 |           0 |           0 |       35846 |           0 |           0 |           0
  average dram access latency (ns)   |       60.78 |         inf |         inf |         inf |       60.74 |         inf |         inf |         inf
  average dram queueing delay        |        7.35 |        0.00 |        0.00 |        0.00 |        7.31 |        0.00 |        0.00 |        0.00
  average dram bandwidth utilization |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
Coherency Traffic                    |             |             |             |             |             |             |             |            
  num loads from dram                |       12021 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from dram cache          |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from remote cache        |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
